test/test_meta.py

```diff
@@ -801,30 +997,32 @@ class TestMeta(TestCase):
     def test_meta(self, device, dtype, op):
         # run the OpInfo sample inputs, cross-referencing them with the
         # meta implementation and check the results are the same.  All
-        # the heavy lifting happens in MetaCrossRefMode
+        # the heavy lifting happens in MetaCrossRefFunctionMode
         func = op.get_op()
```

Comment:
is it worth using the api to get more samples ?

Reply:
I actually wasn't aware how to get more samples, but yes, seems useful?

Reply:
https://github.com/pytorch/pytorch/blob/bdbb7fe37a49cb034773995a4ec192ba5b1b7ba8/torch/testing/_internal/common_methods_invocations.py#L409

https://github.com/pytorch/pytorch/blob/bdbb7fe37a49cb034773995a4ec192ba5b1b7ba8/test/test_ops.py#L313

Reply:
Will do it later